libs/core/__deprecated__/renderer/components/rest/menu/expandable-menu/expandable-menu-subitems.component.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React from "react"
+import { FunctionComponent } from "Core/core/types/function-component.interface"
+import { MenuElementItem } from "Core/__deprecated__/renderer/constants/menu-elements"
+import {
+  LatestSubitemBranchMarker,
+  LatestSubitemBranchMarkerWrapper,
+  SubitemBranchMarkerHorizontalLine,
+  SubitemBranchMarkerVerticalLine,
+  SubitemBranchMarkerWrapper,
+  SubitemButton,
+  SubitemEmptyBox,
+  SubitemWrapper,
+} from "Core/__deprecated__/renderer/components/rest/menu/expandable-menu/expandable-menu-subitems.styles"
+
+const FirstSubitem = () => {
+  return (
+    <SubitemWrapper>
+      <SubitemBranchMarkerWrapper>
+        <SubitemBranchMarkerVerticalLine />
+        <SubitemBranchMarkerHorizontalLine transparent />
+      </SubitemBranchMarkerWrapper>
+      <SubitemEmptyBox />
+    </SubitemWrapper>
+  )
+}
+
+const LatestSubitem: FunctionComponent<MenuElementItem> = (props) => {
+  return (
+    <SubitemWrapper>
+      <LatestSubitemBranchMarkerWrapper>
+        <LatestSubitemBranchMarker />
+      </LatestSubitemBranchMarkerWrapper>
+      <SubitemButton {...props} />
+    </SubitemWrapper>
+  )
+}
+
+const Subitem: FunctionComponent<MenuElementItem> = (props) => {
+  return (
+    <SubitemWrapper>
+      <SubitemBranchMarkerWrapper>
+        <SubitemBranchMarkerVerticalLine />
+        <SubitemBranchMarkerHorizontalLine />
+      </SubitemBranchMarkerWrapper>
+      <SubitemButton {...props} />
+    </SubitemWrapper>
+  )
+}
+
+interface ExpandableMenuSubitemsProps {
+  items: MenuElementItem[]
+}
+
+const ExpandableMenuSubitems: FunctionComponent<
+  ExpandableMenuSubitemsProps
+> = ({ items }) => {
+  if (items.length === 0) {
+    return null
+  }
+
+  if (items.length === 1) {
+    return (
+      <>
+        <FirstSubitem />
+        <LatestSubitem {...items[0]} />
+      </>
+    )
+  }
```

Comment:
Maybe you could call the `FirstSubitem` as `EmptySubitem` or `MarkerSubitem`? It was quite hard for me to understand that there will be only one menu item rendered.

Reply:
https://github.com/mudita/mudita-center/pull/2329/commits/ea2d7b1be85e9ad294442e8429df6259ef14f3bc